npm/src/lib/github.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+
+import GitHub from "github";
+
+const CLIENT = new GitHub({version: "3.0.0"});
+if (process.env.GH_TOK) {
```

Comment:
Does the user have to have GH_TOK set en their env in order for the CLI to work? I've never used the node GH client before.